lib/kasket/query_parser.rb

```diff
@@ -22,6 +23,18 @@ def initialize(model_class)
 
     def parse(sql)
       if match = @supported_query_pattern.match(sql)
+        select = match[1]
+        unless @star_pattern.match? select
+          select_columns = []
+          select.split(/\s*,\s*/).each do |s|
+            return nil unless column_match = @table_and_column_pattern.match(s)
```

Comment:
`nil` not needed and would help it be consistent with the next bare return.

Reply:
Also it would be nice if the comment spelled out what happens if you return nil from here... as I'm not seeing that atm.